tools/export-strings.sh

```diff
@@ -7,8 +7,13 @@ if [ ! -d Blockzilla.xcodeproj ]; then
   exit 1
 fi
 
+if [ -d "focusios-l10n" ]; then
+echo "Focus iOS L10 directory found. Removing to re-clone for fresh start."
+rm -Rf focusios-l10n;
+fi
+
 echo "[*] Cloning mozilla-l10n/focusios-l10n"
-git clone git@github.com:mozilla-l10n/focusios-l10n.git
+git clone https://github.com/mozilla-l10n/focusios-l10n.git
```

Comment:
I'm not aware of Focus's BR setup, but I'll assume this change doesn't impact it.

Reply:
This actually fixes issue for any new person who runs the export and doesn't want to use ssh :)